plugins/eslint/readme.md

```diff
@@ -0,0 +1,38 @@
+# @dotcom-tool-kit/eslint
+
+A plugin to run [eslint](https://eslint.org/) in your JavaScript code. This plugin uses the ESLint Node.js [API](https://eslint.org/docs/latest/developer-guide/nodejs-api) to run ESLint. It loads the ESLint configuration files (.eslintrc.* files) in your repo by default.
+
+## Installation
+
+With Tool Kit [already set up](https://github.com/financial-times/dotcom-tool-kit#installing-and-using-tool-kit), install this plugin as a dev dependency:
+
+```sh
+npm install --save-dev @dotcom-tool-kit/eslint
+```
+
+And add it to your repo's `.toolkitrc.yml`:
+
+```yaml
+plugins:
+    - '@dotcom-tool-kit/eslint'
+```
+
+## Options
+
+| Key | Description | Default value |
+|-|-|-|
+| `files` | The glob patterns for lint target files. This can either be an array of string, or a string | `'**/*.js'` |
```

Comment:
**nitpick**: I've tried to make the description clearer
```suggestion
| `files` | The glob patterns for lint target files. This can either be a string or an array of strings | `'**/*.js'` |
```